### PR TITLE
Streamline deploying process

### PR DESCRIPTION
**Pull request by kaspergrubbe**
- [x] Get rid of colorize,
- [x] Consider to make this compatible with Capistrano v3 or push it to a further release.

### CODE REVIEW

File: lib/simple-capistrano-unicorn/namespace.rb [hunk 1]
```diff
@@ -82,63 +67,122 @@ def clean_old_unicorn(server)
...
+                print ".".colorize(:green)
+              end
+              print "✓\n".colorize(:green).bold
+
+              print ".. code _NOT_ deployed!\n".colorize(:red).bold
+            end
           end
 
-          desc "Restart of Unicorn with downtime"
+          # desc "Restart of Unicorn with downtime"
```
Comment (nj) [inline lib/simple-capistrano-unicorn/namespace.rb]: Why commenting out this?

File: lib/simple-capistrano-unicorn/namespace.rb [hunk 2]
```diff
@@ -82,63 +67,122 @@ def clean_old_unicorn(server)
...
+              old_pid = capture("cat #{unicorn_old_pid}", :hosts => [server]).to_i
+              run "kill -s WINCH #{old_pid}", :hosts => [server]
+              print ".".colorize(:green)
+            end
+            print "✓\n".colorize(:green).bold
+
+            # 3.1 ALL TRAFFIC GOES TO NEW SERVERS, WANT TO KILL OLD?
+            while(true)
+              print "Traffic is handled by new master, are you happy (y/n)? ".colorize(:white)
+              answer = STDIN.gets.downcase.gsub("\n",'')
```
Comment (nj) [inline lib/simple-capistrano-unicorn/namespace.rb]: Probably a good idea to have support for auto-answer, to specify in setting if it always should default to `Y` or ask each time. By the current way, it loosing it's nice, clean and quick deploys.

File: lib/simple-capistrano-unicorn/namespace.rb (hunk 1 above)
Comment (kaspergrubbe) [inline lib/simple-capistrano-unicorn/namespace.rb]: Not sure :)

File: lib/simple-capistrano-unicorn/namespace.rb (hunk 2 above)
Comment (kaspergrubbe) [inline lib/simple-capistrano-unicorn/namespace.rb]: Yeah, need to fix these things :)

**Pull request merged**